frontend/src/metabase/collections/components/BaseItemsTable.jsx

```diff
@@ -68,11 +70,14 @@ BaseItemsTable.propTypes = {
   items: PropTypes.arrayOf(PropTypes.object),
   collection: PropTypes.object,
   selectedItems: PropTypes.arrayOf(PropTypes.object),
+  hasUnselected: PropTypes.bool,
   isPinned: PropTypes.bool,
   renderItem: PropTypes.func,
   sortingOptions: sortingOptsShape,
   onSortingOptionsChange: PropTypes.func,
   onToggleSelected: PropTypes.func,
+  onSelectAll: PropTypes.func,
+  onSelectNone: PropTypes.func,
```

Comment:
Should these props be `isRequired`?

Reply:
There is actually another usage of `BaseItemsTable` for dragging that doesn't need these props, see [here](https://github.com/metabase/metabase/blob/a3c5665b7ec040eaa2f3d090af92930029a6adf9/frontend/src/metabase/containers/dnd/ItemsDragLayer.jsx#L121)